bin/resources/GameIndex.yaml

```diff
@@ -11645,7 +11645,9 @@ SLES-51354:
   region: "PAL-M5"
   compat: 5
   gsHWFixes:
-    mipmap: 1
+    autoFlush: 1 # Fixes effects.
```

Comment:
What effects?

Reply:
A bit hard to describe, but it seems to fix a few minor lighting effects based on distances, I think it's actually very similar to what #6597 does since it does seem to fix a few glow effects, so I'll probably just change it to "glows" instead of "effects".

Reply:
Would be nice to update the comment to mention what it fixes.